jupyter_lidar_inspection.ipynb

```diff
@@ -0,0 +1,918 @@
+    "               n=int(smaller_region_dict['n']),\n",
+    "               s=int(smaller_region_dict['s']),\n",
+    "               w=int(smaller_region_dict['w']),\n",
+    "               e=int(smaller_region_dict['e']),\n",
+    "               flags=\"d\")\n",
+    "gs.run_command(\"g.region\",region='smaller_region')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e356497-1662-443c-9de4-689e139a8c10",
+   "metadata": {},
+   "source": [
+    "## 3D Data Visualization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "755e62d4",
+   "metadata": {},
+   "source": [
+    "Let's do a 3D check of the region: tiledb will manage that our computer survives"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8d4815a0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = os.path.expanduser(\"/your/path/point_cloud.laz\")\n",
+    "array_name = os.path.expanduser(\"/your/path/tiledb_pc\")\n",
+    "filename = \"/your/path/point_cloud.laz\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "114bc70b-a904-4c2f-8331-e585eda9b8e4",
+   "metadata": {},
+   "source": [
+    "Let's create a PDAL pipeline to import our LAZ into a tiledb database."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d84f2b86",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "pipeline = pdal.Reader.las(filename=data).pipeline()\n",
+    "pipeline |= pdal.Writer.tiledb(filename=array_name, x_tile_size=200, y_tile_size=200, z_tile_size=100)\n",
+    "pipeline.execute() # once we create it, we can just load it next time\n",
+    "# The output states the number of points"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aec2215e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pybabylonjs import Show as show\n",
+    "A = tiledb.open(array_name)\n",
+    "df = A.query(attrs=('Red', 'Green','Blue')).df[w:e, s:n, z_min:z_max]\n",
+    "# df = A.query(attrs=('Infrared', 'Green','Red')).df[478000+500:478000+700, 77999+500:77999+700, 406.14:593.856000]\n",
+    "data = {\n",
+    "    'X': df['X'],\n",
+    "    'Y': df['Y'],\n",
+    "    'Z': df['Z'],\n",
+    "    'Red': df['Red'], # or df['Infrared'], df['Green'], df['Red']\n",
+    "    'Green': df['Green'],\n",
+    "    'Blue': df['Blue']\n",
+    "}\n",
+    "\n",
+    "show.point_cloud(source=\"dict\",\n",
+    "                 data=data,\n",
+    "                 point_size = 4,\n",
+    "                 rgb_max=65535,\n",
+    "                 width = 1200,\n",
+    "                 height = 700)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c9aa1f54",
+   "metadata": {},
+   "source": [
+    "- Question 1: What is the difference between a DSM and a DTM?\n",
+    "- Question 2: What resolution, method and filters would you use to generate the DTM and what to generate a DSM with vegetation classes?"
```

Comment:
Either provide a minimal explanation or some hints about (where to search for) the right answer